### ISSUE TITLE
Reusing same directory/repo between benchmarks 

### ISSUE THREAD

**Issue by skshetry** [iterative/dvc-bench#34]

**Comment by pared** [comment]
To sum up the discussion that we had on latest planning meeting:
It seems like it could be useful to create a mechanism for providing `test-repo` in a pre-prepared manner. For example: 
- Test add from https://github.com/iterative/dvc-bench/blob/master/benchmarks/add.py
reuse the same setup. 
- each time benchmark is created, data is copied from known tmp directory
- each time git and dvc is initialized

So, what we could do is to have the whole repo (with git and dvc initialized) prepared, and just copy the whole repo. That could save us a lot of time, especially when we will be creating more advanced benchmarks. 

*Note*
When dealing with pre-added data in repo, we need to remember about running `dvc status` after copying the repo, as inodes of files inside repo will change, and state will need to be updated before benchmark.

**Comment by efiop** [comment]
No longer relevant after #293, it is faster to set things up and run chunky benchmarks in parallel.
